javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -369,6 +369,11 @@ public static AnnotationMirror getSameIgnoringValues(
             new Comparator<AnnotationMirror>() {
                 @Override
                 public int compare(AnnotationMirror a1, AnnotationMirror a2) {
+                    // AnnotationMirror.toString() prints the elements of an annotation in the
```

Comment:
I meant if you have `@IntRange(from=1, to=MAX)` and `@IntRange(to=MAX,from=0)` you would expect `from=0` to come before `from=1`. But because string comparison is used, you get the opposite order.
I don't think we have to fix this here, but a comment might be good. In general, doing things with strings is a hack.